src/components/study-container.js

```diff
@@ -389,7 +393,7 @@ export function StudyContainer({ view, onChangeTab }) {
                     .slice(1)
                     .map((parent) => parent.elementName);
 
-                const studyName = response[0]?.elementName;
+                const studyName = limitChar(response[0]?.elementName, 30);
```

Comment:
i think we don't do that here , there is a file that contain already a method computePageTitle in compute-title.js, just put your modification there